<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 21, 40);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="signingIn">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Neutra Text</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	
	color: rgb(81, 183, 187);
}</string>
      </property>
      <property name="text">
       <string>Signing in ...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <spacer name="right">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="title">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Neutra Text</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF
}</string>
      </property>
      <property name="text">
       <string>omniPhony</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="left">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="circularProgressBarBase">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="circularProgress">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>150</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	
	border-radius: 75px;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.632857 rgba(0, 0, 0, 0), stop:0.633857 rgba(81, 183, 187, 255));
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="circulerBg">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>150</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	border-radius: 75px;
	background-color: rgba(33, 89, 112, 120);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="container">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>15</y>
         <width>130</width>
         <height>130</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	border-radius: 65px;
	background-color: rgb(6, 21, 40);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <zorder>circulerBg</zorder>
      <zorder>circularProgress</zorder>
      <zorder>container</zorder>
     </widget>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
